include/picongpu/plugins/radiation/Radiation.hpp

```diff
@@ -268,7 +284,7 @@ namespace picongpu
                     {
                         writeOpenPMDfile(
                             tmp_result,
-                            restartDirectory + "/" + speciesName + std::string("_radRestart_"));
+                            restartDirectory + "/" + speciesName + std::string("_radRestart"));
```

Comment:
@franzpoeschel Why is this renaming necessary?

Reply:
Because it now depends on the `--radiation.openPMDSuffix` parameter if this underscore is needed: By default, this plugin puts out something like `e_radAmplitudes_700_0_0_0.h5`. With this update, something like `e_radAmplitudes.h5` becomes possible, too – without an underscore. 
So I removed the hardcoded underscore in these two places, and put it instead into the default parameter `  --e_radiation.openPMDSuffix arg (=_%T_0_0_0.bp4)` (compare the parameter `--openPMD.infix _%06T`).

Reply:
Without this change, we would get output like `e_radAmplitudes_.h5`.

Reply:
Okay - thanks for answering - will merge